ui/packages/catalog-ui-search/src/main/webapp/react-component/inputs/enum-input/enum-input.js

```diff
@@ -13,37 +13,51 @@
  *
  **/
 import React, { useState } from 'react'
+
 import Dropdown from '../../dropdown'
-import { Menu, MenuItem } from '../../menu'
+import { Menu, MenuItemDisabled } from '../../menu'
 import TextField from '../../text-field'
 import styled from 'styled-components'
 import { getFilteredSuggestions, inputMatchesSuggestions } from './enumHelper'
 import PropTypes from 'prop-types'
-
+const sources = require('../../../component/singletons/sources-instance')
```

Comment:
<img width="806" alt="Screen Shot 2020-02-15 at 1 26 20 PM" src="https://user-images.githubusercontent.com/11235121/74594738-e4f48280-4ff6-11ea-88e0-7ae725f5f80d.png">

Reply:
Ooh forgot to remove  this after refactoring the source logic out of here thx 👍